site/content/en/docs/manual/basics/create-multi-tasks.md

```diff
@@ -0,0 +1,101 @@
+---
+title: 'Create multi tasks'
+linkTitle: 'Create multi tasks'
+weight: 3
+description: 'Step-by-step guide on how to create and set up multiple tasks'
+---
+
+Use **Create multi tasks** to create multiple video annotation tasks with the same configuration.
+
+> The **Сreate multi tasks** feature is available for videos only.
+
+To create the multi tasks, on the **Tasks** page click **+** and
+select **Create multi tasks**.
+
+![](/images/image254.jpg)
+
+See:
+
+- [Create multi tasks](#create-multi-tasks)
+- [Example](#example)
+- [Errors](#errors)
+- [Advanced configuration](#advanced-configuration)
+
+## Create multi tasks
+
+To add several tasks in one go, open the task configurator:
+
+![Multitack configurator](/images/multitask_configurator.png)
+
+And specify the following parameters:
+
+1. In the **Name** field, enter the name of the new task:
+
+   - Enter the name of the task. If the name includes more than one word, use the underscore: `Word1_word2_word3`
```

Comment:
I'm not sure that the rule make any sense. Should we use `_` instead of a space?